### DISCUSSION TITLE
Tie in game movement with real world movement?

### DISCUSSION THREAD

**Discussion by Generalkidd** [LivingFray/HaloCEVR#90] [Ideas]
So currently if you try to walk around in the real world, it just moves your camera while your actual body stays in the same original position. I was wondering if it's possible to also tie your biped game position to your position in the room? I was looking at the UpdateCamera() function in InputHandler.cpp and was thinking maybe that could be applied to update your biped position in real time but my understanding of all this is limited so I'm not sure if it's actually possible or not. Any thoughts on the possibility of this or perhaps what changes/functions I'd need to add to the code to make that an option?

**Comment by LivingFray** [comment]
Took me far too long to get around to it, but I've added it in 1.4.0 (I feared just moving the biped to the camera position would cause a bunch of clipping issues, but apparently halo handles it fine)
